src/locales/en.json

```diff
@@ -242,6 +242,7 @@
         "bills": "Bills",
         "certified.carbon_copy": "Carbon copy",
         "certified.electronic_safe": "Electronic safe",
+        "coachco2.settings": "CoachCO2 app settings",
```

Comment:
We have `Bank app settings` but also `Photo settings` maybe someday we will have to homogenise those 😛